packages/interledger/src/admin-api/schema.graphql

```diff
@@ -1,22 +1,429 @@
 type Query {
-  accounts: String
+  "Fetch a page of Interledger accounts."
+  ilpAccounts(
+    "Paginating forwards: the cursor before the the requested page."
+    after: String
+    "Paginating backwards: the cursor after the the requested page."
+    before: String
+    "Paginating forwards: The first **n** elements from the page."
+    first: Int
+    "Paginating backwards: The last **n** elements from the page."
+    last: Int
+  ): IlpAccountsConnection!
+
+  "Get an Interledger account by ID."
+  ilpAccount(id: ID!): IlpAccount!
+
+  "Get a trustline by ID."
+  trustline("The ID of the trustline to get." id: ID!): Trustline!
+
+  "Get a webhook by ID."
+  webhook("The ID of the webhook to get." id: ID!): Webhook!
+
+  "Get a deposit by ID."
+  deposit("The ID of the deposit to get." id: ID!): Deposit!
+
+  "Get a withdrawal by ID."
+  withdrawal("The ID of the withdrawal to get." id: ID!): Withdrawal!
+}
+
+type Mutation {
+  "Create Interledger account"
+  createIlpAccount: CreateIlpAccountMutationResponse!
+
+  "Update Interledger account"
+  updateIlpAccount: UpdateIlpAccountMutationResponse!
+
+  "Delete Interledger account"
+  deleteIlpAccount(id: ID!): DeleteIlpAccountMutationResponse!
+
+  "Create Interledger sub-account"
+  createIlpSubAccount(superAccountId: ID!): CreateIlpSubAccountMutationResponse!
+
+  "Transfer between Interledger accounts"
+  transfer(
+    sourceAmount: UInt64!
+    sourceAccountId: ID!
+    destinationAccountId: ID!
+    destinationAmount: UInt64
+    autoCommit: Boolean
+    idempotencyKey: ID!
+  ): TransferMutationResponse
+
+  "Extend Trustline"
+  extendTrustline(
+    "The id of the trustline to extend."
+    trustlineId: ID!
```

Comment:
https://github.com/coilhq/rafiki/pull/51 
>As is, trustlines don't have an id. Trustline methods just take accountId of the sub-account to which the trustline is extended.

Similarly, there's no trustline resource, so the `trustline` query above isn't necessary. Trustline info is just included with (sub)account balances.

Reply:
@wilsonianb Cool, I have updated the schema, please check you're happy with it!